apiserver/allfacades.go

```diff
@@ -197,7 +197,7 @@ func AllFacades() *facade.Registry {
 	)
 
 	reg("Pinger", 1, NewPinger)
-	reg("Provisioner", 3, provisioner.NewProvisionerAPI)
+	reg("Provisioner", 4, provisioner.NewProvisionerAPI)
```

Comment:
You need to keep version 3 around as well.

Reply:
I still use the same type as the change is additive so no new facade type is needed AFAIU.

I will add two entries as follows to have both of them in the facade registry:

reg("Provisioner", 3, provisioner.NewProvisionerAPI)
reg("Provisioner", 4, provisioner.NewProvisionerAPI)